pkg/platformconfig/platformconfig.go

```diff
@@ -34,6 +35,7 @@ type Config struct {
 	AutoScale                AutoScale                `json:"autoScale,omitempty"`
 	CronTriggerCreationMode  CronTriggerCreationMode  `json:"cronTriggerCreationMode,omitempty"`
 	FunctionAugmentedConfigs []LabelSelectorAndConfig `json:"functionAugmentedConfigs,omitempty"`
+	KubeFunctionExposureMode KubeFunctionExposureMode `json:"kubeFunctionExposureMode,omitempty"`
```

Comment:
I wouldn't bind that to `kube`. this behaviour can be served `local` as well (as is user wants to deploy a function, running in a specific network and not expose any port